Sources/AppCoinsSDK/Domain/UseCases/TransactionUseCases.swift

```diff
@@ -17,11 +17,11 @@ internal class TransactionUseCases {
         self.repository = repository
     }
     
-    internal func getTransactionBonus(address: String, package_name: String, amount: String, currency: Coin, completion: @escaping (Result<TransactionBonus, TransactionError>) -> Void) {
+    internal func getTransactionBonus(address: String, package_name: String, amount: String, currency: String, completion: @escaping (Result<TransactionBonus, TransactionError>) -> Void) {
```

Comment:
Use Currency to represent currencies, not String.

Reply:
Done